extensions/amp-video/1.0/autoplay.jss.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createUseStyles} from 'react-jss';
+
+const eq = {
+  pointerEvents: 'none !important',
+  alignItems: 'flex-end',
+  bottom: '7px',
+  height: '12px',
+  opacity: '0.8',
+  overflow: 'hidden',
+  position: 'absolute',
+  right: '7px',
+  width: '20px',
+  zIndex: '1',
+  display: 'flex',
+};
+
+const eqCol = {
+  flex: '1',
+  height: '100%',
+  marginRight: '1px',
+  position: 'relative',
+  '& > div': {
+    animationName: '$eq-animation',
+    animationTimingFunction: 'linear',
+    animationIterationCount: 'infinite',
+    animationDirection: 'alternate',
+    backgroundColor: '#FAFAFA',
+    height: '100%',
+    position: 'absolute',
+    width: '100%',
+    willChange: 'transform',
```

Comment:
FMI, do we have any data on the performance impact of using `will-change`?

Reply:
No actual data. It does have an impact, since it definitely prevents some browsers from re-rendering when changing other CSS props.